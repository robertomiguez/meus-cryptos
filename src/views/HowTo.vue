<template>
    <div class="main">
        <div class="title">Keep track of your cryptos</div>
        <div class="step">First step - Push add button. </div>
        <div><img src="@/assets/step1.jpg"></div>
        <div class="step">Second step - Fill the form</div>
        <div><img src="@/assets/step2.jpg"></div>
        <div class="step">Third step - Crypto list</div>
        <div><img src="@/assets/step3.jpg"></div>
    </div>
</template>

<script>
export default {
  name: 'HowTo'
}
</script>

<style scoped>
.main {
  text-align: center;
  padding-top: 10em;
  font-weight: 100;
  text-align: center;
}
.title {
  font-size: 2em
}
img {
  margin-bottom: 3em;
  width: 25%;
}
.step {
  padding: 1.5em 0;
}
@media screen and (max-width: 600px) {
  .main {
    padding-top: 6em;
  }
  .title {
    font-size: 1.3em
  }
  img {
    width: 90%;
  }
}
</style>
