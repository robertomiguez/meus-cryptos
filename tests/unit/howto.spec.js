import { shallowMount } from '@vue/test-utils'
import HowTo from '@/views/HowTo'

describe('HowTo', () => {
  const build = () => {
    const wrapper = shallowMount(HowTo)

    return {
      wrapper
    }
  }

  it('renders the component', () => {
    // arrange
    const { wrapper } = build()
    // assert
    expect(wrapper.html()).toMatchSnapshot()
  })
})
