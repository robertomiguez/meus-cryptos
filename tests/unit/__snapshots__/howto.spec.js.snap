// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HowTo renders the component 1`] = `
<div class="main">
  <div class="title">Keep track of your cryptos</div>
  <div class="step">First step - Push add button. </div>
  <div><img src="@/assets/step1.jpg"></div>
  <div class="step">Second step - Fill the form</div>
  <div><img src="@/assets/step2.jpg"></div>
  <div class="step">Third step - Crypto list</div>
  <div><img src="@/assets/step3.jpg"></div>
</div>
`;
